ansible/roles/slapd/files/etc/ldap/schema/debops/ldapns.schema

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: GPL-2+
```

Comment:
DebOps uses the `reuse` Python package to check validity of all files with the https://reuse.software/ project. All files require standardized set of Copyright and License information. You can install the `reuse` Python package and run `reuse lint` to see which files are not correctly marked.

The `GPL-2+` string is probably not a correct one, it would probably be `GPL-2.0-or-later`.

It would be great to check the source repositories and find out the author(s) of these files and approximate ranges of when they maintained them.

Reply:
My guess is that `ldapns.schema` is copied directly from [fusiondirectory-schema] package. [the schema] is [listed] as GPL-2+ in the COPYRIGHT file for that Debian package.

  [fusiondirectory-schema]: https://packages.debian.org/buster/all/fusiondirectory-schema/filelist
  [the schema]: https://salsa.debian.org/debian/fusiondirectory/-/blob/master/debian/copyright?ref_type=heads#L1120
  [listed]: https://salsa.debian.org/debian/fusiondirectory/-/blob/master/debian/copyright?ref_type=heads#L1412-1415

Reply:
Yes, but the file still needs to reflect that in a way that `reuse` can parse. That's why we need to change the format from `GPL-2+` to `GPL-2.0-or-later`, and add Copyright information. The reason is, the file now comes from the DebOps repository itself, not from an outside package.

Reply:
Ok, I've done some archaeology, dug up some authors, and updated the SPDX headers accordingly...